src/mappings/ContributionReward.ts

```diff
@@ -0,0 +1,94 @@
+import 'allocator/arena'
+export { allocate_memory }
+
+import { Entity, Address, U256, Bytes, Value, store, crypto, ByteArray } from '@graphprotocol/graph-ts'
+
+import { NewContributionProposal, RedeemReputation, RedeemEther, RedeemNativeToken, RedeemExternalToken } from '../types/ContributionReward/ContributionReward'
+
+import { concat, updateRedemption } from '../utils'
+
+export function handleNewContributionProposal(event: NewContributionProposal): void {
+    let accountId = crypto.keccak256(concat(event.params._beneficiary, event.params._avatar)).toHex()
```

Comment:
why ? is that redundant here ?

Reply:
This was missing the account entry. I just updated it. I am creating account for all users and then member accounts will be the ones with reputation

Reply:
should you just add that to the proposals entry ? The proposal and crProposals .